app/components/BreadcrumbNav.tsx

```diff
@@ -0,0 +1,40 @@
+import {
+  Breadcrumb,
+  BreadcrumbBar,
+  BreadcrumbLink,
+} from '@trussworks/react-uswds'
+
+export default function BreadcrumbNav({
+  path,
+  pathPrepend,
+  className,
+}: {
+  path: string
+  pathPrepend?: string
+  className?: string
+}) {
+  const [lastBreadcrumb, ...breadcrumbs] = path.split('/').reverse()
+  const sortedCrumbs = breadcrumbs.reverse()
+
+  return (
+    <BreadcrumbBar className={className}>
+      <>
+        {...sortedCrumbs.map((item) => {
+          const crumbPath: string = sortedCrumbs
+            .slice(0, sortedCrumbs.indexOf(item) + 1)
+            .join('/')
+          return (
+            <Breadcrumb key={item}>
+              <BreadcrumbLink href={[pathPrepend, `${crumbPath}`].join('/')}>
+                {item}
+              </BreadcrumbLink>
+            </Breadcrumb>
+          )
+        })}
```

Comment:
Here's a simpler approach:

```jsx
let cumulativePath = prependPath
...
return (
  <BreadcrumbBar>
    {breadcrumbs.map((breadcrumb) => {
      cumulativePath = `${cumulativePath}/${breadcrumb}`
      return (
        <Breadcrumb key={cumulativePath}>
          <BreadcrumbLink href={cumulativePath}>
            {breadcrumb}
          </BreadcrumbLink>
        </Breadcrumb>
      )
    })}
    <Breadcrumb>
      <BreadcrumbLink href={`${cumulativePath}/${lastBreadcrumb}`}>
        ${lastBreadcrumb}
      </BreadcrumbLink>
    </Breadcrumb>
  </BreadcrumbBar>
)
```

Also, note that `item` is not a valid key because it is possible for the path to contain two path components that are equal strings, hence not unique.